execution_engine/src/core/engine_state/engine_config.rs

```diff
@@ -1,22 +1,48 @@
-const DEFAULT_MAX_QUERY_DEPTH: u64 = 5;
+use crate::shared::{system_config::SystemConfig, wasm_config::WasmConfig};
+
+pub const DEFAULT_MAX_QUERY_DEPTH: u64 = 5;
+
+pub const DEFAULT_WASMLESS_TRANSFER_COST: u32 = 10_000;
```

Comment:
Do we need this to be defined here though? I think I've originally put it in protocol_data.rs - what about moving it to where it is used which would be: `execution_engine/src/shared/system_config.rs`

Reply:
Moved to `system_config`